2-mandatory/3-stocks.js

```diff
@@ -34,8 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
-}
+    let averageArray = [];
+    for(let elementS of CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS){
```

Comment:
It would be better to use the closingPricesForAllStocks parameter as the input instead of CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS,it's not a good practice to use a global variable within a function unless it's necessary. Other than that, all good